packages/sdk/src/react/ui/modals/CreateListingModal/Modal.tsx

```diff
@@ -102,13 +113,25 @@ const Modal = observer(() => {
 		);
 	}
 
-	if (collectableIsError || collectionIsError) {
+	if (collectableIsError || collectionIsError || currenciesIsError) {
+		return (
+			<ErrorModal
+				isOpen={createListingModal$.isOpen.get()}
+				chainId={Number(chainId)}
+				onClose={createListingModal$.close}
+				title="List item for sale"
+			/>
+		);
+	}
+
+	if (!currencies || currencies.length === 0) {
 		return (
 			<ErrorModal
 				isOpen={createListingModal$.isOpen.get()}
 				chainId={Number(chainId)}
 				onClose={createListingModal$.close}
 				title="List item for sale"
+				message="No ERC-20s are configured for the marketplace, contact the marketplace owners"
```

Comment:
We can place listings in Native currencies, just not orders

```suggestion
				message="No currencies are configured for the marketplace, contact the marketplace owners"
```